cmd/libs/go2idl/openapi-gen/generators/openapi.go

```diff
@@ -517,3 +627,191 @@ func (g openAPITypeWriter) generateSliceProperty(t *types.Type) error {
 	g.Do("},\n},\n},\n", nil)
 	return nil
 }
+
```

Comment:
Validation doesn't seem tightly coupled to openapi - why not just generate it separately (and find ways to reuse more of the boilerplate above)?  Is there an advantage to having the two together that I missed, or going forward are we saying that everything we generate that starts from our core model objects should be logically associated with openapi (defaulters, validation, clients, etc)?